integr8ly/integreatly-operator
# MGDAPI-6221 update marin3r to 0.13.0

# Issue link
https://issues.redhat.com/browse/MGDAPI-6412

# What
Update Marin3r to 0.13.0

# Verification steps
Fresh install covered by e2e.

Upgrade scenario verification:
1. Install RHOAM 1.40.0 from this index: quay.io/mstoklus/managed-api-service-index:1.40.0
2. Once installed apply RHOAM 1.41.0 index: quay.io/mstoklus/managed-api-service-index:1.41.0
3. Confirm upgrade was successful
4. Confirm upgrade of marin3r from 0.12.3 to 0.13.0 was successful 
5. Navigate to marin3r rate limiting config map and edit the value to 5 (before doing so, scale down rhoam operator)
6. Restart rate limit pods
7. Ping 3scale 6 times, rate limiting should work.
